src/react-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nypl/design-system-react-components",
-  "version": "0.0.7",
+  "version": "0.0.10",
```

Comment:
We'll need to figure out how to make new releases automatically tag and push. Backlog it goes.